pkg/controller/controller.go

```diff
@@ -431,8 +440,20 @@ func genLabels(tfjobKey string) map[string]string {
 // When a pod is added, set the defaults and enqueue the current tfjob.
 func (tc *TFJobController) addTFJob(obj interface{}) {
 	tfjob := obj.(*tfv1alpha2.TFJob)
-	log.Infof("Adding tfjob: %s", tfjob.Name)
+	msg := fmt.Sprintf("TFJob %s is created.", tfjob.Name)
+	log.Info(msg)
 	scheme.Scheme.Default(tfjob)
+
+	// Leave a created condition and set StartTime.
+	newTFJob := tfjob.DeepCopy()
+	now := metav1.Now()
+	newTFJob.Status.StartTime = &now
+	err := tc.updateTFJobConditions(newTFJob, tfv1alpha2.TFJobCreated, tfJobCreatedReason, msg)
```

Comment:
LGTM but maybe I will change the code here for test purpose. Handler based implementation is more test friendly

Reply:
Oh my fault, LGTM without any comment.

Reply:
Oh my fault, LGTM without any comment.